src/operators/operators.cpp

```diff
@@ -662,11 +620,10 @@ void OperatorBase<dim,nstate,real>::get_FR_correction_parameter (
             k = 4.24e-7;
         k/=2.0;//since orthonormal
         k/=pow(pow(2.0,curr_cell_degree),2);//since ref elem [0,1]
-//        k /= pow((2.0*curr_cell_degree+1.0)*(pow(pfact*cp,2)),dim -1.0);//for multiple dim tensor product
     }
 }
-template <int dim, int nstate, typename real>
```

Comment:
If you're copy-pasting, there is a better way.

Below, you can use if(dim >= 1), and if (dim>=2) to avoid copying code.

Reply:
How has this been fixed?

Reply:
Sorry I don't know which snippet this is in reference to? All places where I check if(dim ==) have different chunks.
I think this refers to build_local_Flux_Reconstruction_operator, but the chunks that look similar call different differential operators.

Reply:
Hahaha GitHub is starting to crash because your PR is so big. Can't link the code anymore.


![Screenshot from 2022-04-22 13-29-16](https://user-images.githubusercontent.com/9758507/164764945-07fc5a25-c410-488a-9df7-1e576b1db7ad.png)

Reply:
Lol you mean github is suggesting that the PR gets accepted ;)
Ahh I see, ok yes I am fixing this as we speak.

Reply:
fixed

Reply:
@dougshidong looks like this has been addressed